ebs/ebs.go

```diff
@@ -504,6 +504,50 @@ func (d *Driver) MountPoint(req Request) (string, error) {
 	return volume.MountPoint, nil
 }
 
+func (d *Driver) GetVolumesInfo(ids []string) ([]map[string]string, error) {
+	d.mutex.Lock()
```

Comment:
What is the lock protecting ? It's quite broad and includes the call to ebs.
If it's just the loading of the files, consider unlocking before the getVolumes (I'm not sure if it buys a lot).